src/main/java/frc/robot/joystickManager/JoystickState.java

```diff
@@ -0,0 +1,10 @@
+package frc.robot.joystickManager;
+
+public enum JoystickState {
```

Comment:
the word state is unclear perhaps use something more informative like bindings

Reply:
Bindings are the actual button configs (there is a file called JoystickBindings), and states are like pointers telling the joystick what bindings are active now

Reply:
I agree binding isnt ideal but states  is just really unclear try to fiind a better name for it